ansible/playbooks/AdoptOpenJDK_Unix_Playbook/trss.yml

```diff
@@ -6,6 +6,9 @@
 - hosts: all
   user: root
   become: yes
+  vars:
+    TRSS_working_dir: "/home/jenkins/openjdk-test-tools/TestResultSummaryService"
+    TRSS_config_path: "/data/db/credentials/trssConf.json"
```

Comment:
Do we create this file `/data/db/credentials/trssConf.json`? If not, we will get error when starting TRSS.

Reply:
I would recommend manually creating and populating the trssConf.json file before running the playbook. How do you feel about it?

Reply:
I suppose we could create a template with a password of "password" ... Depends if there's a' risk of that making it into production - is that a password that's used during the creation of the database or will it just throw a (reasonably easy to debug) error message if the passwords don't match?